### ISSUE TITLE
[Feature request] Add games with score

### ISSUE THREAD

**Issue by inson1** [fent/pokecry#2]
I mean there could be x types of games - 10 pokemons, 20, 50 100, all pokemons in generation/s, custom (textbox)
with selected difficulty #4 
and generation/s
and switch if pokemons can reappear (if all pokemons in generation/s then dont ask)

**Comment by fent** [comment]
yeah i was thinking of adding some sort of leaderboard at some point, maybe one for each generation, and for all combined
